### ISSUE TITLE
Unsuccessful logins

### ISSUE THREAD

**Issue by bqv** [bqv/slide#4]
```
login isn’t working.
Android: 12 SKQ1.211006.001 MIUI 13.0.1
    Do you have any more detail on that - is it just not doing anything, or does it throw an error, or crash
Nope. No errors or crashes. It looks like it’s working, with the line going across the top, but then that stops. I can click the arrow to start again but the same thing happens. If I go back I see I’m not logged in.
    (or see what’s actually happening in logcat if you know how)
I do not know how.
    Also, how long is your password; that’s been a known issue in all apps some cases
Now this sounds suspicious! It’s 10 characters long, for now.
```

c. https://feddit.uk/comment/456855 and parents

**Comment by bqv** [comment]
May be case sensitivity

**Comment by matteganau** [comment]
I had the same issue and making the password shorter (50 > 30) seemed to do the trick.

I could not find anywhere to update the password in Slide. I tried signing in as a Guest and then adding the same account again with the shorter password but it would not sign in (same issue as with longer password) so I uninstalled + reinstalled.

Slide is the only reddit app I ever used so thanks for the quick port and good luck!
